Makefile

```diff
@@ -227,3 +227,15 @@ $(patsubst %,build/%.go,$(BENCHMARKS)): build/%.go: %.py $(COMPILER)
 $(BENCHMARK_BINS): build/benchmarks/%_benchmark: build/benchmarks/%.go $(RUNTIME) $(STDLIB)
 	@mkdir -p $(@D)
 	@go build -o $@ $<
+
+# ------------------------------------------------------------------------------
+# Installation
+# ------------------------------------------------------------------------------
+
+install: $(RUNNER_BIN) $(COMPILER) $(RUNTIME) $(STDLIB)
+	install -Dm755 build/bin/grumpc $(DESTDIR)/usr/bin/grumpc
+	install -Dm755 build/bin/grumprun $(DESTDIR)/usr/bin/grumprun
+	install -d $(DESTDIR)/usr/lib/go
+	cp -rv --no-preserve=ownership build/lib $(DESTDIR)/usr/
```

Comment:
Actually I don't think we want to install everything in lib since it includes some stuff like pylint that isn't actually required for grumpy. Basically all we want is what's in site-packages/grumpy. And that should end up in the pythonpath for the selected PYTHON bin... Can we get the prefix for that Python and then put it in lib in there?